React/my-app/src/screens/book_detail/layout.js

```diff
@@ -19,50 +20,53 @@ import {
 } from "./strings.js";
 import strings from "./strings";
 
-const Book_detail = ({ image, title, author, year, genre }) => (
-  <div className="mainContainer">
-    <NavBarContainer />
-    <Link className="back" to="/">
-      {VOLVER}
-    </Link>
-    <div className="center">
-      <div className="info">
-        <img className="main_book" src={image || bigDefaultImg} />
-        <div className="description_box">
-          <h1 className="book_title">{title}</h1>
-          <h1 className="other_info">{author}</h1>
-          <h1 className="other_info">{year}</h1>
-          <h1 className="other_info">{genre}</h1>
-          <p className="description">{TEXTO}</p>
-          <button className="rent_button" type="button">
-            {ALQUILAR}
-          </button>
+const Book_detail = ({ book, loading }) =>
+  loading ? (
+    <RingLoader />
+  ) : (
+    <div className="mainContainer">
+      <NavBarContainer />
+      <Link className="back" to="/">
+        {VOLVER}
+      </Link>
+      <div className="center">
+        <div className="info">
+          <img className="main_book" src={book.image_url || bigDefaultImg} />
+          <div className="description_box">
```

Comment:
I think that you can clear a little this code if you create specifics components.
For example, `descriptionBox Component` `suggests Component`.
Create this components into the components file in the same folder
```
youScreen
    -components
             -subComponent1
                    -index.js
    - index.js
    - layout.js
```